course/views.py

```diff
@@ -106,8 +106,24 @@ def get(self, request, *args, **kwargs):
         elif lti_msg:
             messages.info(request, html.escape(lti_msg))
 
-        return super().get(request, *args, **kwargs)
+        later_instance = None
+
+        if self.instance.is_past:
+            course = self.instance.course
```

Comment:
I think there is also `self.course`. If so, then I think this "alias" might not be so important. Same consideration can be used for the `now`, which is used only once.

Reply:
There seemed to be that alias, so I cleared that now too